charts/kong/templates/deployment.yaml

```diff
@@ -23,6 +23,7 @@ spec:
   replicas: {{ .Values.replicaCount }}
   {{- end }}
   {{- end }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
```

Comment:
Since that is a deployment specific field, I believe this should be nested under `deployment`.

Reply:
okay, I will update it.